src/Components/NodeSymbol/NodeSymbol.tsx

```diff
@@ -1,13 +1,30 @@
 import React from "react";
 import { Node } from "Models/Node";
 import Symbol from "Components/Symbol/Symbol";
-import { siC, siCplusplus, siPython } from "simple-icons/icons";
+import { siC, siCplusplus, siPython, siGo, siJavascript, siHtml5, siCss3, siJson, siMarkdown, siTypescript, siReact, siGnubash, siKotlin, siSass, siYarn, siBabel, siGit, siPhp, siRuby, siScala  } from "simple-icons/icons";
```

Comment:
Is the autoformatter "on"? I think that the line shouldn't be that long with it enabled